contributing.md

```diff
@@ -0,0 +1,67 @@
+Contributing
+============
+
+## Issue Tracking
+
+New feature requests, changes, enhancements, non-methodology features, and bug reports can be filed as new issues in the
+[GitHub.com issue tracker](https://github.com/entralliance/entr_runtime/issues) at any time. Please be sure to fully describe the
+issue.
+
+<!---For other issues, please email the ENTR distribution list at `xx`.--->
+
+## Repository
+
+The entr_runtime repository is hosted on GitHub, and located here: https://github.com/entralliance/entr_runtime
+
+This repository is organized using a modified git-flow system. Branches are organized as follows:
+
+- master: Stable release version. Must have good test coverage and may not have all the newest features.
+- dev: Development branch which contains the newest features. Tests must pass, but code may be unstable.
+- feature/xxx: Branch from dev, should reference a GitHub issue number.
+
+To work on a feature, please fork entr_runtime first. Then clone your fork locally when you are ready to get to work. If your git status is unclean after you clone  (e.g., lots of untracked files and changes), chances are you are dealing with a Unicode normalization issue (specifically, the Unicode decomposition of filenames under macOS). This can be an issue on project with contributors using a mix of MacOS, Linux, and Windows. To resolve it, delete your clone. Then issue `git config --global core.precomposeunicode true` and clone your fork again.
```

Comment:
I've run into the Windows vs. Unix line-ending issue (autocrlf git setting) a lot before; I always have to google it to fix it, but it could be worth adding to troubleshooting guidelines; on that note, it may be worth pushing this "what to do when running into clone issues" guidance into a dedicated troubleshooting or FAQ section